### PR TITLE
fix/syscoin-receive

### PR DESCRIPTION
**Pull request by leozera14**
- [x] Fix syscoin receive loading bug 
- [x] Fix loading zIndex value, the old one was making the loading be in front at menus.
- [x] Fix activeAddress error that was in console application at the DappController

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "@babel/runtime": "^7.14.0",
     "@headlessui/react": "^1.6.0",
     "@heroicons/react": "^1.0.5",
-    "@pollum-io/sysweb3-core": "^1.0.21",
+    "@pollum-io/sysweb3-core": "1.0.19",
```
Comment (claudiovb) [inline package.json:48]: If possible change sysweb3-core version

File: source/pages/Receive/Receive.tsx [hunk 2]
```diff
@@ -25,21 +26,28 @@ export const Receive = () => {
...
 
   useEffect(() => {
+    const abortController = new AbortController();
+
     const setNewAddress = async () => {
-      if (
-        (activeNetwork.chainId === 57 || activeNetwork.chainId === 5700) &&
-        (await controller.wallet.account.sys.setAddress())
-      ) {
+      if (activeNetwork.chainId === 57 || activeNetwork.chainId === 5700) {
```
Comment (claudiovb) [inline source/pages/Receive/Receive.tsx:32]: This verification can lead to potential bugs, these chainId are the same for sysoin UTXO and NEVM mainnet and testnet. 
Should have an extra validation to guarantee we're just calling this function on the UTXO case. (Probably using isBitcoinBasedChain would be enough)

**Review by claudiovb — CHANGES_REQUESTED** [review]
Needs to guarantee that validations that need only to be done on UTXO chains are only done on UTXO chains. Probably the isBitcoinBasedNetwork is more reliable for that than using network.chainId;

The function being called inside setAddress from SyscoinController is getLatestUpdateforAccount:
https://github.com/syscoin/pali-wallet/blob/189b59b6c6b84d596dd69b1dbf8a4e25c66a1e7a/source/scripts/Background/controllers/account/syscoin.ts#L180
It fetches way more information than needed it would be ideal to have a simple function that just fetches the new receivingAddress and resolve the promise leaving getLatestUpdateforAccount to run async later this will greatly improve the loading time for this page.

File: package.json (hunk 1 above)
Comment (leozera14) [reply package.json]: It's not possible, the new version is not working for some functionalities

**Review by claudiovb — APPROVED** [review]

**Pull request merged**